externs/shaka/player.js

```diff
@@ -720,6 +720,8 @@ shaka.extern.ManifestConfiguration;
  *   startAtSegmentBoundary: boolean,
  *   smallGapLimit: number,
  *   jumpLargeGaps: boolean,
+ *   gapThreshold: number,
+ *   gapJumpingPollInterval: number,
```

Comment:
Include documentation for what these are and include entries in the demo.  See 6c012aa4678b6f4be0b3c04e9c16b99543886b53 for an example.